yarn-project/sequencer-client/src/sequencer/sequencer.ts

```diff
@@ -150,7 +151,7 @@ export class Sequencer {
 
       // Get l1 to l2 messages from the contract
       this.log('Requesting L1 to L2 messages from contract');
-      const l1ToL2Messages = this.takeL1ToL2MessagesFromContract();
+      const l1ToL2Messages = await this.consumePendingL1ToL2Messages();
```

Comment:
Maybe this should just be get, consume infers removal from the pending messages queue.